app/models/results.js

```diff
@@ -92,6 +92,19 @@ const ResultSchema = new Schema({
 });
 
 ResultSchema.set('toObject', {virtuals: true});
+ResultSchema.set('toJSON', {
```

Comment:
toJSON is native method - it's not good idea to overwrite it.. there is already feature in mongoose-query to ignore certain keys like __id and _id when needed if I remember coreectly..